test/unit/math/src/sincos.cpp

```diff
@@ -32,8 +32,8 @@ class SinCosTestKernel
         // (PTX kernel (float) was just empty)
         FP check_sin = alpaka::math::sin(acc, arg);
         FP check_cos = alpaka::math::cos(acc, arg);
-        FP result_sin = 0.;
-        FP result_cos = 0.;
+        FP result_sin = FP{0};
+        FP result_cos = FP{0};
```

Comment:
```suggestion
        auto result_sin = FP{0};
        auto result_cos = FP{0};
```

Only if the CI fails again.